styleguide.css

```diff
@@ -0,0 +1,273 @@
+
+@font-face {
+    font-family: 'vremena_groteskregular';
+    src: url('./fonts/vremenagrotesk-webfont.woff2') format('woff2'),
+    url('./fonts/vremenagrotesk-webfont.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+@font-face {
+    font-family: 'vremena_groteskbold';
+    src: url('./fonts/vremenagroteskbold-webfont.woff2') format('woff2'),
+    url('./fonts/vremenagroteskbold-webfont.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+@font-face {
+    font-family: 'vremena_groteskbold_italic';
+    src: url('./fonts/vremenagroteskbolditalic-webfont.woff2') format('woff2'),
+    url('./fonts/vremenagroteskbolditalic-webfont.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+@font-face {
+    font-family: 'vremena_groteskbook';
+    src: url('./fonts/vremenagroteskbook-webfont.woff2') format('woff2'),
+    url('./fonts/vremenagroteskbook-webfont.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+@font-face {
+    font-family: 'vremena_groteskbook_italic';
+    src: url('./fonts/vremenagroteskbookitalic-webfont.woff2') format('woff2'),
+    url('./fonts/vremenagroteskbookitalic-webfont.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+@font-face {
+    font-family: 'vremena_groteskitalic';
+    src: url('./fonts/vremenagroteskitalic-webfont.woff2') format('woff2'),
+    url('./fonts/vremenagroteskitalic-webfont.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+@font-face {
+    font-family: 'vremena_groteskmedium';
+    src: url('./fonts/vremenagroteskmedium-webfont.woff2') format('woff2'),
+    url('./fonts/vremenagroteskmedium-webfont.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+@font-face {
+    font-family: 'vremena_groteskmedium_italic';
+    src: url('./fonts/vremenagroteskmediumitalic-webfont.woff2') format('woff2'),
+    url('./fonts/vremenagroteskmediumitalic-webfont.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+
+/* Sporting grotesque */
+
+@font-face {
+    font-family: 'sporting_grotesque';
+    src: url('./fonts/sporting_grotesque_gras-webfont.woff2') format('woff2'),
+    url('./fonts/sporting_grotesque_gras-webfont.woff') format('woff');
+    font-weight: bold;
+    font-style: normal;
+}
+@font-face {
+    font-family: 'sporting_grotesque';
+    src: url('./fonts/sporting_grotesque_normal-webfont.woff2') format('woff2'),
+    url('./fonts/sporting_grotesque_normal-webfont.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+
+/* General */
+
+body,
+button,
+input,
+select,
+optgroup,
+textarea {
+    color: #00283C;
+    font-family: 'vremena_groteskregular', sans-serif;
+    font-size: 19px;
+    line-height: 1.5;
+
+}
+
+/* Titles */
+
+h1 {
+    font-size:42px;
+    margin: 40px 0;
+  }
+
+h2{
+    font-size:32px;
+    margin: 20px 0 40px;
+}
+
+h3{
+	font-size:28px;
```

Comment:
Indent is larger here.
We usually work with 2 spaces for indent. What is your policy regarding this?